sdk/identity/Azure.Identity/CHANGELOG.md

```diff
@@ -9,6 +9,8 @@
 ### Bugs Fixed
 
 ### Other Changes
+- Marked `UsernamePasswordCredential` as obsolete because Resource Owner Password Credentials (ROPC) token grant flow is incompatible with multifactor authentication (MFA), which Microsoft Entra ID requires for most tenants. See
```

Comment:
```suggestion
- Marked `UsernamePasswordCredential` as obsolete because Resource Owner Password Credentials (ROPC) token grant flow is incompatible with multifactor authentication (MFA), which Microsoft Entra ID requires for all tenants. See
```